django_afip/clients.py

```diff
@@ -11,6 +13,12 @@
 
 TZ_AR = pytz.timezone(pytz.country_timezones["ar"][0])
 CIPHERS = DEFAULT_CIPHERS + "HIGH:!DH:!aNULL"
```

Comment:
Looks like you're missing a leading `:` when you concatenate DEFAULT_CIPHERS and "HIGH:!DH:!aNULL".
You're appending "HIGH" to the last cipher spec from the defaults, so you're messing up the last cipherstring in the list:

```py
>>> (DEFAULT_CIPHERS + "HIGH:!DH:!aNULL")[-20:]
':!MD5HIGH:!DH:!aNULL'
>>> # !MD5HIGH is not a valid spec, so you're invalidating the original !MD5
```

It still works because the only thing really needed for AFIP at the moment is to block Diffie–Hellman because of the weak DH group they have configured. This means `:!DH` is all you need, I think. Also `!aNULL` is already in the defaults, and `HIGH` is kind of redundant, because you're appending to the defaults, not creating an independent cipherlist.

```py
CIPHERS = DEFAULT_CIPHERS + ":!DH"
```